payments/honorarium.md

```diff
@@ -0,0 +1,29 @@
+# OLS Honoraria Policy
+
+This policy covers how OLS gives honoraria. How OLS employees, independent contractors, or consultants receive honoria from other parties and engagement will be detailed in a separate policy.
+
+An honorarium is a one-time, token payment granted in recognition of an individual’s contribution to OLS’s work. 
+
+The types of activities for which honoraria may be offered include:
+- Mentoring, speaking, and facilitation in an OLS cohort
+- Participation in other OLS events as a trainer, speaker, mentor, etc
+- Participation in other OLS activities, e.g. hiring
+- Participation/contribution in an event on behalf of OLS
+
+No honoraria from OLS can be offered to currently engaged OLS employees, independent contractors, or consultants. The recipient of the honorarium is not permitted to set the honorarium amount. The arrangement between the recipient and OLS should not involve an existing contract. 
+
+Currently, OLS may offer:
+- 250 USD for expert speakers in an OLS cohort;
+- 500 USD for mentoring in an OLS cohort;
+- 50 USD/hour, for facilitation in an OLS cohort;
+
+Generally, OLS community members who have been asked to perform a paid task by OLS community organisers are paid at 50 USD/hour.
+
+Individuals who are offered an honorarium can choose to donate the honorarium back to OLS’s work or to a charity of their choice. They can also choose to ask OLS to pay for a training session/conference fee up to the value of the payment - this offer expires in a year after the submission of the honorarium payment form, if unused.
```

Comment:
this offer expires in a year after the submission -> this offer expires one year (=12 months) after the submission

Rewording might make it more clear?